README.md

```diff
@@ -61,6 +61,42 @@ Implementation details
   * Optional runtime blinding which attempts to frustrate differential power analysis.
   * The precomputed tables add and eventually subtract points for which no known scalar (secret key) is known, preventing even an attacker with control over the secret key used to control the data internally.
 
+Obtaining and verifying
+-----------------------
+
+The git tag for each release (e.g. `v0.6.0`) is GPG-signed by one of the maintainers.
+For a fully verified build of this project, it is recommended to obtain this repository
+via git, obtain the GPG keys of the signing maintainer(s), and then verify the release
+tag's signature using git.
+
+This can be done with the following steps:
+
+1. Obtain the GPG keys listed in [SECURITY.md](./SECURITY.md).
+2. If possible, cross-reference these key IDs with another source controlled by its owner (e.g.
+   social media, personal website).
+    - This is to mitigate the unlikely case that incorrect content is being presented by this repository.
```

Comment:
if you touch this again,  no need to use a list: 
```suggestion
2. If possible, cross-reference these key IDs with another source controlled by its owner (e.g. social media, personal website).
   This is to mitigate the unlikely case that incorrect content is being presented by this repository.
```

Reply:
Fixed